Cython/Compiler/ExprNodes.py

```diff
@@ -5807,7 +5807,8 @@ def generate_result_code(self, code):
                         and self.type.is_int
                         and self.type.signed
                         and self.function.result() in ('abs', 'labs', '__Pyx_abs_longlong')):
-                        goto_error = 'if (unlikely(%s < 0)) { PyErr_SetString(PyExc_OverflowError, "value too large"); %s; }' % (self.result(), code.error_goto(self.pos))
+                        goto_error = 'if (unlikely(%s < 0)) { PyErr_SetString(PyExc_OverflowError, "value too large"); %s; }'
```

Comment:
My preferred way to cut such a line is
```
"long %string with %s" % (
    arg1, arg2, ...)
```
Also for long function call lines etc. Having a line end with `(` makes the indentation in the next line very obvious.